src/imio/esign/browser/actions.py

```
@@ -32,14 +33,16 @@ def index(self):
         if not signers:
             return self._finished(failed_msgid="Could not get signers to add to the session!")
         # watchers = self.get_watchers()
-        add_files_to_session(
+        session_id, _session = add_files_to_session(
```

Ruff (0.15.6)

[warning] Unpacked variable `session_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)